expr/func_removeemptyseries_test.go

```diff
@@ -5,21 +5,31 @@ import (
 	"testing"
 
 	"github.com/grafana/metrictank/api/models"
+	"github.com/grafana/metrictank/schema"
```

Comment:
This file was added since it merges in Sean's work in https://github.com/grafana/metrictank/pull/1823/files